xml/ns-System.Xml.xml

```diff
@@ -167,7 +167,7 @@
   
 <a name="security"></a>   
 ## Security considerations  
- The types and members in the <xref:System.Xml> namespace rely on the . The following sections discuss security issues that are specific to XML technologies. For details, see the specific classes and members mentioned, and visit the [XML Developer Center](http://go.microsoft.com/fwlink/?linkid=42458) for technical information, downloads, newsgroups, and other resources for XML developers.  
+ The types and members in the <xref:System.Xml> namespace rely on the [.NET Framework security system](~/docs/standard/security/index.md). The following sections discuss security issues that are specific to XML technologies. For details, see the specific classes and members mentioned, and visit the [XML Developer Center](http://go.microsoft.com/fwlink/?linkid=42458) for technical information, downloads, newsgroups, and other resources for XML developers.  
```

Comment:
we should really change that section to cover more than just framework. Should we call it Security in .NET?

Reply:
one more thing: that developer center doesn't exist anymore. Remove it?